packages/static-wado-creator/lib/writer/WriteMultipart.js

```diff
@@ -0,0 +1,12 @@
+const { v4: uuid } = require('uuid');
+
+const WriteMultipart = async (writeStream, contentType, content) => {
```

Comment:
Can you pass a headers object that gets written instead of a contentType?  There are other header values we may eventually need.